static/js/components/instrument/InstrumentForm.jsx

```diff
@@ -9,36 +9,81 @@ import { dataUriToBuffer } from "data-uri-to-buffer";
 import { showNotification } from "baselayer/components/Notifications";
 import { submitInstrument } from "../../ducks/instrument";
 import { modifyInstrument } from "../../ducks/instrument";
+import { fetchFollowupApis } from "../../ducks/followupApis";
 import { fetchInstruments } from "../../ducks/instruments";
 
 const InstrumentForm = ({ onClose, instrumentID = null }) => {
   const { instrumentList } = useSelector((state) => state.instruments);
   const { telescopeList } = useSelector((state) => state.telescopes);
   const { enum_types } = useSelector((state) => state.enum_types);
+  const [followupApis, setFollowupApis] = useState([]);
+  const [formData, setFormData] = useState({});
   const dispatch = useDispatch();
 
-  const handleSubmit = async ({ formData }) => {
-    const keys = Object.keys(formData);
-    if (keys.includes("field_data") && formData.field_data) {
-      const parsed_field_data = dataUriToBuffer(formData.field_data);
-      formData.field_data = new TextDecoder().decode(parsed_field_data.buffer);
+  useEffect(() => {
```

Comment:
hmmm this seems odd. I feel like the followupApis only need to be hydrated in the ducks and then refreshed based on websocket calls. Why are they fetched in the component here?